embedded-graphics/src/mock_display.rs

```diff
@@ -416,19 +416,21 @@ where
         let pattern_height = pattern.len();
         assert!(
             pattern_width <= SIZE,
-            "Test pattern must not be wider than {}",
+            "Test pattern must not be wider than {} columns",
             SIZE
         );
         assert!(
             pattern_height <= SIZE,
-            "Test pattern must not be taller than {}",
+            "Test pattern must not be taller than {} rows",
             SIZE
         );
-        for row in pattern {
+        for (row_idx, row) in pattern.iter().enumerate() {
             assert_eq!(
                 row.len(),
                 pattern_width,
-                "Every row in the pattern must be {} characters wide",
+                "Row #{} is {} characters wide instead of {}",
```

Comment:
The new message doesn't mention that every row needs to be the same width, so that the user can't easily tell where the `instead of X` value comes from.

Reply:
That's a fair point but I wasn't able to come up with a better alternative. Repeating "characters wide" really looks weird and my creativity was/is limited :)

Reply:
My submission if you need some inspiration:

```
Row #{} is {} characters wide (must be {} characters to match all other rows).
```

Reply:
Thanks, that was helpful. I modified it a bit, since MockDisplay panics on the first incorrect row and so we can't say the others after it are correct :)